Sources/Sentry/SentryLevelMapper.m

```diff
@@ -2,33 +2,56 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@implementation SentryLevelMapper
+NSString *const kSentryLevelNameNone = @"none";
+NSString *const kSentryLevelNameDebug = @"debug";
+NSString *const kSentryLevelNameInfo = @"info";
+NSString *const kSentryLevelNameWarning = @"warning";
+NSString *const kSentryLevelNameError = @"error";
+NSString *const kSentryLevelNameFatal = @"fatal";
 
-+ (SentryLevel)levelWithString:(NSString *)string
+SentryLevel
+levelWithString(NSString *string)
 {
-    if ([string isEqualToString:SentryLevelNames[kSentryLevelNone]]) {
+    if ([string isEqualToString:kSentryLevelNameNone]) {
         return kSentryLevelNone;
     }
-    if ([string isEqualToString:SentryLevelNames[kSentryLevelDebug]]) {
+    if ([string isEqualToString:kSentryLevelNameDebug]) {
         return kSentryLevelDebug;
     }
-    if ([string isEqualToString:SentryLevelNames[kSentryLevelInfo]]) {
+    if ([string isEqualToString:kSentryLevelNameInfo]) {
         return kSentryLevelInfo;
     }
-    if ([string isEqualToString:SentryLevelNames[kSentryLevelWarning]]) {
+    if ([string isEqualToString:kSentryLevelNameWarning]) {
         return kSentryLevelWarning;
     }
-    if ([string isEqualToString:SentryLevelNames[kSentryLevelError]]) {
+    if ([string isEqualToString:kSentryLevelNameError]) {
         return kSentryLevelError;
     }
-    if ([string isEqualToString:SentryLevelNames[kSentryLevelFatal]]) {
+    if ([string isEqualToString:kSentryLevelNameFatal]) {
         return kSentryLevelFatal;
     }
 
     // Default is error, see https://develop.sentry.dev/sdk/event-payloads/#optional-attributes
     return kSentryLevelError;
 }
 
-@end
+NSString *
+nameForLevel(SentryLevel level)
```

Comment:
m: Isn't this too generic for a global symbol?

I thing something like this would be more appropriate:

```suggestion
NSString *
nameForSentryLevel(SentryLevel level)
```

or make it static

Same goes for other functions like nameForCategory, categoryForString...